css/types/image.json

```diff
@@ -632,6 +631,39 @@
                 }
               }
             },
+            "premultiplied_gradients": {
+              "__compat": {
+                "description": "Gradients applied to pre-multiplied color space (prevents grays from appearing in gradients with transparency)",
+                "support": {
+                  "chrome": {
+                    "version_added": "29"
+                  },
+                  "chrome_android": "mirror",
+                  "edge": "mirror",
+                  "firefox": {
+                    "version_added": "36"
```

Comment:
I left a comment about this on the associated issue for this PR: https://github.com/mdn/browser-compat-data/issues/14598#issuecomment-1853114556. It turns out Firefox doesn’t support premultiplied alpha in gradients correctly even in the latest version of Firefox. So this entry is incorrect. Here’s the link to the Firebox bug I filed for this, which has been confirmed: https://bugzilla.mozilla.org/show_bug.cgi?id=1869681.

Reply:
I am noticing a difference in the gradient in Firefox vs. other browsers using the test code you provided in the bug:

<img width="2056" alt="image" src="https://github.com/mdn/browser-compat-data/assets/5179191/1828a52a-a1b0-466b-89fa-4d75e6901f7d">

(Top-Left: Chrome, Bottom-Left: Safari, Top-Right: Firefox)